src/Containers.jsx

```diff
@@ -160,10 +172,19 @@ const ContainerActions = ({ con, container, onAddNotification, localImages, upda
 
     const actions = [];
     if (isRunning || isPaused) {
-        // TODO: cockpit-podman currently isn't aware of podman quadlets as they don't keep containers around when stopped, failed or not yet started by default.
-        // Allowing a user to stop or restart a container can result in the container disappearing from the list without a way to start it again. Until cockpit-podman
-        // can list quadlets these actions are disabled.
-        if (!isSystemdService) {
+        // Allow restarting quadlets from the logged in user and superuser
+        if (isSystemdService && [0, null].includes(container.uid)) {
+            actions.push(
+                <DropdownItem key="stop"
+                          onClick={() => stopQuadletContainer()}>
+                    {_("Stop")}
+                </DropdownItem>,
+                <DropdownItem key="restart"
+                          onClick={() => restartQuadletContainer()}>
+                    {_("Restart")}
+                </DropdownItem>,
+            );
+        } else if (!isSystemdService) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2286-9d63ad0b-20251017-133332-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)